test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/TestHelpers/IWebHostPortExtensions.cs

```diff
@@ -10,24 +10,11 @@ namespace Microsoft.AspNetCore.Hosting
 {
     public static class IWebHostPortExtensions
     {
-        public static string GetHost(this IWebHost host, bool isHttps = false)
```

Comment:
GetHost and GetPort weren't added to Testing, so we are just removing these outright?